src/OpenLoco/src/Input/MouseInput.cpp

```diff
@@ -1347,6 +1347,7 @@ namespace OpenLoco::Input
                 break;
 
             case Ui::WidgetType::panel:
+            case Ui::WidgetType::newsPanel:
```

Comment:
Maybe we should not add this line, given the window can't/shouldn't be repositioned or resized anyway.

Reply:
I've added it just in case, in the future this will most likely vanish, the resize bar should be its own widget and not rely on the frame type.